how-to-guides/arabica-devnet.md

```diff
@@ -175,7 +175,6 @@ There are multiple explorers you can use for Arabica:
 
 - [https://arabica.celenium.io](https://arabica.celenium.io)
 - [https://explorer.celestia-arabica-11.com](https://explorer.celestia-arabica-11.com)
-- [https://celestiascan.com](https://celestiascan.com)
```

Comment:
wonder if this is down forever or just for now?

Reply:
cc @lzrscg

Reply:
yes its down indefinitely, best to remove for now

Reply:
confirmed.